music_assistant/providers/plex/player_remote.py

```diff
@@ -0,0 +1,1696 @@
+            LOGGER.info(
+                f"Created PlexServer for '{self.player_name}' with client ID: {self.client_id}"
+            )
+
+            self.server = PlexRemoteControlServer(
+                plex_provider=self.plex_provider,
+                port=self.port,
+                client_id=self.client_id,
+                ma_player_id=self.ma_player_id,
+            )
+            LOGGER.info(
+                f"Remote control server for '{self.player_name}' bound to MA player: "
+                f"{self.ma_player_id}"
+            )
+
+            await self.server.start()
+
+            # Step 4: Start GDM broadcasting
+            self.gdm = PlexGDMAdvertiser(
+                instance_id=self.client_id,
+                port=self.port,
+                name=self.player_name,
+                product="Music Assistant",
+                version=self.plex_provider.mass.version
+                if self.plex_provider.mass.version != "0.0.0"
+                else "1.0.0",
+            )
+            self.gdm.start()
+
+            LOGGER.info(f"Player '{self.player_name}' is now discoverable on port {self.port}")
+
+    async def stop(self) -> None:
+        """Stop this player's remote control."""
+        if self.remote_control:
+            if self.gdm:
+                await self.gdm.stop()
+
+            if self.server:
+                await self.server.stop()
+
+            LOGGER.info(f"Stopped remote control for player '{self.player_name}'")
+
+
+class PlexRemoteControlServer:
+    """HTTP server to receive Plex remote control commands."""
+
+    def __init__(
+        self,
+        plex_provider: PlexProvider,
+        port: int = 32500,
+        client_id: str | None = None,
+        ma_player_id: str | None = None,
+    ) -> None:
+        """Initialize remote control server."""
+        self.provider = plex_provider
+        self.plex_server = plex_provider._plex_server
+        self.port = port
+        self.client_id = client_id or plex_provider.instance_id
+        self.app = web.Application()
+        self.subscriptions: dict[str, dict[str, object]] = {}
+        self.runner: web.AppRunner | None = None
+        self.http_site: web.TCPSite | None = None
+
+        # Play queue tracking (Plex-specific state that doesn't exist in MA)
+        self.play_queue_id: str | None = None
+        self.play_queue_version: int = 1
+        # Map queue index to item ID
+        self.play_queue_item_ids: dict[int, int] = {}
+
+        # Track MA queue state to detect when we need to sync to Plex
+        self._last_synced_ma_queue_length: int = 0
+        self._last_synced_ma_queue_keys: list[str] = []
+
+        # Specific MA player this server controls (set by PlayerRemoteInstance)
+        self._ma_player_id = ma_player_id
+
+        # Shared HTTP client for timeline broadcasts
+        self._http_client: aiohttp.ClientSession | None = None
```

Comment:
let's try to always use the shared hrtp client that lives in mass.http_session and mass.http_session_no_ssl